.env

```diff
@@ -4,15 +4,16 @@ MONGO_URL=mongodb://127.0.0.1:27017/mua-system-accounts
 SECRETS=verylongsecret1 verylongsecret2
 APP_URL=http://accounts.emailfox.link/
 API_URL=http://api.emailfox.link
-STATIC_SERVER_URL=http://localhost:10007/
-STATIC_SERVER_PORT=10007
+TEST_STATIC_SERVER_URL=http://localhost:10007/
```

Comment:
Here as well. Don't use two separate variables for the same info (port) because it's misleading.